CMakeLists.txt

```diff
@@ -12,25 +12,8 @@ find_package(Boost REQUIRED system filesystem)
 find_package(ASSIMP QUIET)
 if (NOT ASSIMP_FOUND)
   find_package(PkgConfig REQUIRED)
-  pkg_check_modules(ASSIMP assimp)
-endif()
-if (ASSIMP_FOUND)
```

Comment:
im working on the merged repo as we speak ;-)

until kinetic is fully ready, i guess i agree with you we should keep updating indigo